modules/serverless-kn-func-delete.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies
 
-// * /serverless/cli_reference/kn-func-ref.adoc
+// * serverless/cli_tools/kn-func-ref.adoc
 
 :_content-type: PROCEDURE
-[id="delete-function-kn_{context}"]
+[id="serverless-kn-func-delete_{context}"]
 = Deleting a function
 
-You can delete a function from your cluster by using the `kn func delete` command.
+You can delete a function from your cluster by using the `kn func delete` command. This is useful when a function is no longer a part of your solution. By deleting a function, you can save resources and keep your cluster and solution cleaner.
```

Comment:
Not sure what the value is of the additional text here really, sort of feels like fluff.

Reply:
Well since the docs guidelines call for explicit Why, here's the Why. The Why for deleting something is obvious, and hence feels like fluff. -_o_-

Reply:
[Used your suggestion]